bower.json

```diff
@@ -1,22 +1,25 @@
 {
   "name": "device-orientation",
-  "description": "Polymer element to detecting device orientation",
+  "description": "Polymer element to detecting device orientation and motion",
   "authors": [
     "Abdón Rodríguez Davila <a@abdonrd.com>"
   ],
   "keywords": [
     "web-component",
+    "webcomponent",
     "polymer",
     "device",
+    "position",
     "orientation",
+    "motion",
     "sensor"
   ],
-  "license": "https://abdonrd.github.io/LICENSE.txt",
+  "license": "MIT",
   "repository": {
     "type": "git",
     "url": "git://github.com/abdonrd/device-orientation.git"
   },
-  "main": "device-orientation.html",
+  "main": "device-orientation-all.html",
```

Comment:
Done. But I get `bower invalid-meta  The "main" field has to contain only 1 file per filetype; found multiple .html files: ["device-motion.html","device-orientation.html"]` warning.